migrations/v5.js

```diff
@@ -0,0 +1,248 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Resources - v5.2.1 to v5.3.0', async () => {
+  let course, contentObjects;
+  whereFromPlugin('Resources - from v5.2.1', { name: 'adapt-contrib-resources', version: '<5.3.0' });
+
+  whereContent('Resources - where Resources', async (content) => {
+    course = getCourse();
+    return course?._resources;
+  });
+
+  mutateContent('Resources - add _resourcesItems _isGlobal attribute', async (content) => {
+    if (course._resources._resourcesItems?.length) {
+      course._resources._resourcesItems.forEach((item) => {
+        item._isGlobal = true;
+      });
+    }
+    return true;
+  });
+
+  mutateContent('Resources - add content object _resources object', async (content) => {
+    contentObjects = content.filter(({ _type }) => _type === 'page');
+    contentObjects.forEach((contentObject) => {
+      contentObject._resources = {};
+    });
+    return true;
+  });
+
+  mutateContent('Resources - add _isEnabled attribute to content object', async (content) => {
+    contentObjects.forEach((contentObject) => {
+      contentObject._resources._isEnabled = true;
+    });
+    return true;
+  });
+
+  mutateContent('Resources - add _resourcesItems to content object', async (content) => {
+    contentObjects.forEach((contentObject) => {
+      contentObject._resources._resourcesItems = [];
+    });
+    return true;
+  });
+
+  checkContent('Resources - check _resourcesItems _isGlobal attribute', async (content) => {
+    if (course._resources._resourcesItems?.length) {
+      const isValid = course._resources._resourcesItems.every((item) => _.has(item, '_isGlobal'));
+      if (!isValid) throw new Error('Resources - _resourcesItems _isGlobal does not exist');
+    }
+    return true;
+  });
+
+  checkContent('Resources - check content object _resources object', async (content) => {
+    const isValid = contentObjects.every((contentObject) => _.has(contentObject, '_resources'));
+    if (!isValid) throw new Error('Resources - content object _resources does not exist');
+    return true;
+  });
+
+  checkContent('Resources - check _isEnabled attribute', async (content) => {
+    const isValid = contentObjects.every((contentObject) => _.has(contentObject, '_resources._isEnabled'));
+    if (!isValid) throw new Error('Resources - _isEnabled does not exist');
```

Comment:
Clarify that this is for content objects
```suggestion
    if (!isValid) throw new Error('Resources - content objects _isEnabled does not exist');
```